server/engine/src/main/resources/resourceBundles/messages_en.properties

```diff
@@ -24,6 +24,7 @@ CompilerDirectivesTransformation.sequenceNumber=The first character of the seque
 ContinuationLineTransformation.compilerDirectiveContinued=Compiler directives cannot be continued on another line
 ContinuationLineTransformation.continuationLineContentAreaA=A continuation line cannot contain values in the Content Area A
 ContinuationLineTransformation.periodRequired=IGYDS1082-E A period was required.
+missing.period=A period was assumed before "%s".
```

Comment:
What does "assumed" mean? That it was missing in the code but the parser added it anyway?

Reply:
Yes. The parser assumed it, went ahead to parse the code ahead, instead of just failing at that point.

Reply:
Ah so if it's a copy of the compiler message that's fine